gecko-media/gecko/glue/include/gfx2DGlue.h

```diff
@@ -0,0 +1 @@
+// FIXME: This is an ampty header to avoid moving WebMDemuxer.cpp to glue/
```

Comment:
Maybe upstream's WebMDemuxer.cpp could not include gfx2DGlue.h?

Reply:
Hum yes, it's worth a try :)